crates/core/src/api/invocation.rs

```diff
@@ -74,27 +68,36 @@ impl ApiInvocation {
 		api: &ApiDefinition,
 		body: ApiBody,
 	) -> Result<Option<(ApiResponse, ResponseInstruction)>> {
-		let (action, action_config) =
-			match api.actions.iter().find(|x| x.methods.contains(&self.method)) {
-				Some(v) => (&v.action, &v.config),
-				None => match &api.fallback {
-					Some(v) => (v, &None),
-					None => return Ok(None),
-				},
-			};
-
-		let mut configs: Vec<&ApiConfig> = Vec::new();
-		let global = ctx.tx().get_db_optional_config(opt.ns()?, opt.db()?, "api").await?;
-		configs.extend(global.as_ref().map(|v| v.inner.try_into_api()).transpose()?);
-		configs.extend(api.config.as_ref());
-		configs.extend(action_config);
+		// TODO: Figure out if it is possible if multiple actions can have the same method, and if
```

Comment:
Just one method can be defined at a time. I believe that the fallback action is never used now tho right?

Reply:
Yeah we could change it, let's do that in a followup PR